projects/core/src/lib/datatable/table/table.component.html

```diff
@@ -66,5 +66,12 @@
         </tr>
     </table>
 </div>
-
-<mat-paginator [pageSizeOptions]="[5, 10, 20]" showFirstLastButtons></mat-paginator>
+<mat-form-field id="columnSelector" class="dropdown">
+    <mat-label>Select columns</mat-label>
+    <mat-select multiple [(ngModel)]="displayedColumns">
+        <mat-option *ngFor="let columnName of checkboxLabels" [value]="columnName" (click)="showCheckboxColumn()">
+            {{ fixedColumnDescriptors(columnName) }}
+        </mat-option>
+    </mat-select>
+</mat-form-field>
+<mat-paginator class="paginate" [pageSizeOptions]="[5, 10, 20]" showFirstLastButtons></mat-paginator>
```

Comment:
why do you need the paginate class?

Reply:
Right, I adressed the mat-paginator directly inside the css, that's cleaner 75328626bcc97511d5f50b686668d3dbb8dfaa6c